src/main/java/org/jenkinsci/plugins/urltrigger/URLTriggerCause.java

```diff
@@ -9,9 +9,18 @@
 
     public static final String NAME = "URLTrigger";
     public static final String CAUSE = "A change within the response URL invocation";
+    public static String urlTrigger;
+
+    public static void SetUrlTrigger(String url) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
thanks @ellieMayVelasquez!  I resolved your comment.